src/Form/StrawberryRunnersToolsForm.php

```diff
@@ -152,37 +150,8 @@ public function callJmesPathprocess(array &$form, FormStateInterface $form_state
       $result = $exception->getMessage();
     }
 
-    $response->addCommand(new \Drupal\strawberry_runners\Ajax\UpdateCodeMirrorCommand('#jmespathoutput', json_encode($result,JSON_PRETTY_PRINT)));
+    $response->addCommand(new UpdateCodeMirrorCommand('#jmespathoutput', json_encode($result,JSON_PRETTY_PRINT)));
 
     return $response;
   }
-
-  /**
-   * Limit access to the Tools according to their restricted state.
-   *
-   * @param \Drupal\Core\Session\AccountInterface $account
-   *   The account object.
-   * @param int $node
-   *   The node id.
-   */
-  public function accessTools(AccountInterface $account, $node) {
```

Comment:
Just making sure that the remove is intentional

Reply:
Yes (its Monday so I may be mistaken?). Was not used (checked whole code base). Will refactor the route properly and rewrite it (was also using undefined classes so quite bad)